### DISCUSSION TITLE
Releases

### DISCUSSION THREAD

**Discussion by tjsuominen** [LawnchairLauncher/lawnchair#3423] [Q&A] [answered]
Curious, are there eventually going to be new releases (Play Store)? If yes, any timeframe when those could be expected?

**Comment by flying-night-slasher** [comment]
Same here like post the most stable version of 11/12 then work on 13 then release 13 to play when development begin's on 14 or something because I'm using lawn chair 2 (play store) as my daily driver but it's slow to open and I don't really do beta stuff.

**Comment by SuperDragonXD** [comment, marked as answer]
Hello! I know that this reply is a bit late, but please [read our FAQ](https://lawnchair.app/faq/#when-is-the-next-update) for info regarding our roadmap. We also have released both Lawnchair and Lawnicons in Google Play as well.
